x-pack/solutions/security/plugins/elastic_assistant/server/lib/attack_discovery/schedules/fields/field_map.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FieldMap, alertFieldMap } from '@kbn/alerts-as-data-utils';
+import {
+  ALERT_ATTACK_DISCOVERY_ALERTS_CONTEXT_COUNT,
+  ALERT_ATTACK_DISCOVERY_ALERT_IDS,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG_ACTION_TYPE_ID,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG_CONNECTOR_ID,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG_DEFAULT_SYSTEM_PROMPT_ID,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG_MODEL,
+  ALERT_ATTACK_DISCOVERY_API_CONFIG_PROVIDER,
+  ALERT_ATTACK_DISCOVERY_DETAILS_MARKDOWN,
+  ALERT_ATTACK_DISCOVERY_ENTITY_SUMMARY_MARKDOWN,
+  ALERT_ATTACK_DISCOVERY_MITRE_ATTACK_TACTICS,
+  ALERT_ATTACK_DISCOVERY_REPLACEMENTS,
+  ALERT_ATTACK_DISCOVERY_SUMMARY_MARKDOWN,
+  ALERT_ATTACK_DISCOVERY_TITLE,
+  ALERT_ATTACK_DISCOVERY_USERS,
+  ALERT_ATTACK_DISCOVERY_USER_ID,
+  ALERT_ATTACK_DISCOVERY_USER_NAME,
+} from './field_names';
+
+export const attackDiscoveryAlertFieldMap: FieldMap = {
+  /**
+   * Default alert-as-data fields
+   */
+  ...alertFieldMap,
+
+  /**
+   * Attack discovery fields
+   */
+  [ALERT_ATTACK_DISCOVERY_USERS]: {
```

Comment:
nit: Consider ordering the entries in this file alphabetically

Reply:
done in https://github.com/elastic/kibana/pull/215184/commits/05cc112bd7260691402f301864bacdc2cf1d340b